tests/test_unit.py

```diff
@@ -117,38 +117,15 @@ def test_collect_package_data() -> None:
     stubs = Path("data") / "test_typeshed" / "stubs"
     pkg_data = collect_package_data(stubs / "singlefilepkg")
     assert pkg_data.top_level_packages == ["singlefilepkg-stubs"]
-    assert pkg_data.package_data == (
-        {"singlefilepkg-stubs": ["__init__.pyi", "METADATA.toml", "py.typed"]}
-    )
+    assert pkg_data.package_data == ({"singlefilepkg-stubs": ["METADATA.toml"]})
```

Comment:
Tests were checking for strings in `package_data`, but I don't think there's any test to ensure that the right files are *actually* collected.

Reply:
This would be especially important to test if we swap build backends (https://github.com/typeshed-internal/stub_uploader/issues/166)